config/settings/local_untracked.py.dist

```diff
@@ -42,7 +42,7 @@ import os
 # postgres DB config
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
         'NAME': 'seed',
         'USER': 'your-username',
         'PASSWORD': 'your-password',
```

Comment:
@adrian-lara we need the MAPQUEST api key setting right? probably worth adding that to the local untracked and instructions (I thought it was but now I don't see it).